File: 0268-missing-number/0268-missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        
        r = len(nums)
        b = [i for i in range(r + 1)]  # Full range from 0 to n
        for num in nums:
            b.remove(num)  # Safely remove each number that exists in nums
        return b[0] 


        

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int k = 0;
        for (int i = 0 ; i< nums.length -1;i++){
            for (int j = 0 ; j < nums.length-i-1;j++){
                if (nums[j]>nums[j+1]){
                int temp = nums[j];
                nums[j]=nums[j+1];
                nums[j+1]=temp;
                }

            }
        }
        for(int i =0; i<nums.length;i++){
            if( i != nums[i]){
                return i;
                // break;
                
            }
           
            
        }
        // return k ;
        return nums.length;
    }
}